datasets/indic_glue/indic_glue.py

```diff
@@ -0,0 +1,899 @@
+    }
+        """
+    ),
+    "md": textwrap.dedent(
+        """
+        @inproceedings{Dhanwal2020AnAD,
+        title={An Annotated Dataset of Discourse Modes in Hindi Stories},
+        author={Swapnil Dhanwal and Hritwik Dutta and Hitesh Nankani and Nilay Shrivastava and Y. Kumar and Junyi Jessy Li and Debanjan Mahata and Rakesh Gosangi and Haimin Zhang and R. R. Shah and Amanda Stent},
+        booktitle={LREC},
+        year={2020}
+        }
+        """
+    ),
+    "wiki-ner": textwrap.dedent(
+        """
+        @inproceedings{pan-etal-2017-cross,
+        title = "Cross-lingual Name Tagging and Linking for 282 Languages",
+        author = "Pan, Xiaoman  and
+        Zhang, Boliang  and
+        May, Jonathan  and
+        Nothman, Joel  and
+        Knight, Kevin  and
+        Ji, Heng",
+        booktitle = "Proceedings of the 55th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)",
+        month = jul,
+        year = "2017",
+        address = "Vancouver, Canada",
+        publisher = "Association for Computational Linguistics",
+        url = "https://www.aclweb.org/anthology/P17-1178",
+        doi = "10.18653/v1/P17-1178",
+        pages = "1946--1958",
+        abstract = "The ambitious goal of this work is to develop a cross-lingual name tagging and linking framework for 282 languages that exist in Wikipedia. Given a document in any of these languages, our framework is able to identify name mentions, assign a coarse-grained or fine-grained type to each mention, and link it to an English Knowledge Base (KB) if it is linkable. We achieve this goal by performing a series of new KB mining methods: generating {``}silver-standard{''} annotations by transferring annotations from English to other languages through cross-lingual links and KB properties, refining annotations through self-training and topic selection, deriving language-specific morphology features from anchor links, and mining word translation pairs from cross-lingual links. Both name tagging and linking results for 282 languages are promising on Wikipedia data and on-Wikipedia data.",
+    }
+        """
+    ),
+}
+
+_TEXT_FEATURES = {
+    "wnli": {"hypothesis": "sentence1", "premise": "sentence2"},
+    "copa": {"premise": "premise", "choice1": "choice1", "choice2": "choice2", "question": "question"},
+    "sna": {"text": "text", "label": "label"},
+    "csqa": {"question": "question", "answer": "answer", "category": "category", "title": "title"},
+    "wstp": {
+        "sectionText": "sectionText",
+        "correctTitle": "correctTitle",
+        "titleA": "titleA",
+        "titleB": "titleB",
+        "titleC": "titleC",
+        "titleD": "titleD",
+        "url": "url",
+    },
+    "inltkh": {"label": "label", "text": "text"},
```

Comment:
Could you use ClassLabel for the `label` field of inltkh, bbca, iitp-mr, and iitp-pr ?

Reply:
I missed these earlier. Made the changes